livekit/_utils.py

```diff
@@ -22,3 +24,90 @@ async def get(self) -> T:
             await self._event.wait()
         self._event.clear()
         return self._queue.popleft()
+
+
+class Queue(asyncio.Queue[T]):
+    def __init__(self,
+                 loop: Optional[asyncio.AbstractEventLoop] = None,
+                 maxsize: int = 0) -> None:
+        super().__init__(maxsize)
+        self._loop = loop or asyncio.get_event_loop()
```

Comment:
Am I right to assume self._loop will be used for the asyncio.Queue?

Reply:
how would it know to use self._loop?

Reply:
this class inherits asyncio.Queue, which also use self._loop.

Reply:
ok, I don't think it's safe to override loop in that way.. 

https://github.com/python/cpython/blob/b88d9e75f68f102aca45fa62e2b0e2e2ff46d810/Lib/asyncio/mixins.py#L9

Reply:
right

Reply:
it is only being overrided when not used but I guess I could try to find a better solution